influxdb_store_test.go

```diff
@@ -10,6 +10,10 @@ import (
 )
 
 const (
+	clientEventKey             string = schemaPrefix + clientEventSchema
+	clientEventSchema          string = "HTTPClient"
+	serverEventKey             string = schemaPrefix + serverEventSchema
+	serverEventSchema          string = "HTTPServer"
```

Comment:
Could we inline all of these strings instead @chris-ramon ? I think it would make the tests more readable